printing_auto_base/models/printing_auto.py

```diff
@@ -0,0 +1,132 @@
+# Copyright 2022 Jacques-Etienne Baudoux (BCIM) <je@bcim.be>
+# Copyright 2022 Michael Tietz (MT Software) <mtietz@mt-software.de>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import base64
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError, ValidationError
+from odoo.osv import expression
+from odoo.tools.safe_eval import safe_eval
+
+
+class PrintingAuto(models.Model):
```

Comment:
docstring explaining what this thing does?

Reply:
done